app/middlewares/auth.js

```diff
@@ -24,3 +24,11 @@ exports.isAuthorized = (req, _, next) => {
     return next(error);
   }
 };
+
+exports.isAuthorizedByRoles = (...permittedRoles) => (req, res, next) => {
+  const { headers } = req;
+  const token = getToken(headers.authorization);
+  const payloadToken = verifyToken(token);
+  if (payloadToken && permittedRoles.includes(payloadToken.role)) return next();
+  return res.status(403).json({ message: 'Forbidden' });
```

Comment:
Here is better to create a forbidden error in errors.js and send it in the next, something like:

`return next(forbiddenError('User not authorized'));`

Reply:
forbidden error applied